docs/key_import.md

```diff
@@ -0,0 +1,290 @@
+# Overview
+
+This document serves as an introductory point for users interested in reducing
+their hot-wallet risks, allowing them to maintain on-chain funds outside of
+`lnd` but still be able to manage them within `lnd`. As of `v0.13.0-beta`, `lnd`
+is able to import BIP-0049 and BIP-0084 extended public keys either at the
+account path (`m/purpose'/coin_type'/account'`) or at the address index path
+(`m/purpose'/coin_type'/account'/change/address_index`) as watch-only through
+the `WalletKit` APIs.
+
+Note that in order to follow the rest of this document and/or use the
+`WalletKit` APIs, users will need to obtain a `lnd` build compiled with the
+`walletrpc` tag. Our release builds already include this tag by default, so this
+would only be necessary when compiling from source.
+
+# `lnd`'s Default Wallet Accounts
+
+Upon initializing `lnd`, a wallet is created with four default accounts:
+
+* A custom BIP-0049 account (more on this later) to generate NP2WKH external
+  addresses.
+* A BIP-0084 account to generate P2WKH external and change addresses.
+* A catch-all BIP-0049 account where all imported BIP-0049 address keys (NP2WKH
+  addresses) exist within.
+* A catch-all BIP-0084 account where all imported BIP-0049 address keys (P2WKH
+  addresses) exist within.
+
+Prior to `v0.13.0-beta`, these accounts were abstracted away from users. As part
+of the key import feature, they are now exposed through the new `WalletKit` RPCs
+(`ListAccounts`, `ImportAccount`, `ImportPublicKey`) and the `lncli wallet
+accounts` command.
+
+```shell
+$ lncli wallet accounts
+NAME:
+   lncli wallet accounts - Interact with wallet accounts.
+
+USAGE:
+   lncli wallet accounts command [command options] [arguments...]
+
+COMMANDS:
+     list           Retrieve information of existing on-chain wallet accounts.
+     import         Import an on-chain account into the wallet through its extended public key.
+     import-pubkey  Import a public key as watch-only into the wallet.
+
+OPTIONS:
+   --help, -h  show help
+```
+
+## Account Details
+
+Before interacting with the new set of APIs, users will want to become familiar
+with how wallet accounts are represented within `lnd`. The
+`WalletKit.ListAccounts` RPC or `lncli wallet accounts list` command can be used
+to retrieve the details of accounts.
+
+```shell
+$ lncli wallet accounts list
```

Comment:
👍   nice touch with all the example commands, will likely make it much easier for users to following along and grok how things are set up